elixir-cocktails/phoenix/creating-forms-with-no-database-table/post.md

```diff
@@ -0,0 +1,171 @@
+# Creating form in Phoenix without database table
+
+## The problem
+
+Suppose you want to have some form in your app. It could be a contact form, or some search form.
+That form will have no connection with any database table, but you want to provide some validation on fields.
```

Comment:
some (...) some (...) some (...) can we improve the style a bit here, @amaranthrose ?